docs/api/weights/sar.csv

```diff
@@ -2,6 +2,9 @@ Weight,Channels,Source,Citation,License
 ResNet50_Weights.SENTINEL1_GRD_DECUR, 2,`link <https://github.com/zhu-xlab/DeCUR>`__,`link <https://arxiv.org/abs/2309.05300>`__,"Apache-2.0"
 ResNet50_Weights.SENTINEL1_GRD_MOCO, 2,`link <https://github.com/zhu-xlab/SSL4EO-S12>`__,`link <https://arxiv.org/abs/2211.07044>`__,"CC-BY-4.0"
 ResNet50_Weights.SENTINEL1_GRD_SOFTCON, 2,`link <https://github.com/zhu-xlab/softcon>`__,`link <https://arxiv.org/abs/2405.20462>`__,"CC-BY-4.0"
+Swin_V2_B_Weights.SENTINEL1_MI_SATLAS,2,`link <https://github.com/allenai/satlas>`__,`link <https://arxiv.org/abs/2211.15660>`__,ODC-BY
+Swin_V2_B_Weights.SENTINEL1_SI_SATLAS,2,`link <https://github.com/allenai/satlas>`__,`link <https://arxiv.org/abs/2211.15660>`__,ODC-BY
+Unet_Weights.UMBRA_GEC_OPENEARTHMAP_SAR, 1,`link <https://github.com/cliffbb/DFC2025-OEM-SAR-Baseline/>`__,`link <https://arxiv.org/abs/2501.10891>`__,"CC-BY-4.0"
```

Comment:
Where can I find this license documented?

Reply:
The dataset is CC-BY-4.0 licensed so the model is also the same license found here https://registry.opendata.aws/umbra-open-data/

Reply:
@cliffbb can you clarify here what the license you are releasing the model weights under?